services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -1292,10 +1295,40 @@ static void emitPushJobStatusMetrics(
    */
   @Override
   public void sendPushJobDetails(PushJobStatusRecordKey key, PushJobDetails value) {
+    sendPushJobDetailsToLocalRT(key, value);
+    if (isParent()) {
+      String lastDualWriteError = "";
+      for (Map.Entry<String, ControllerClient> entry: getControllerClientMap(getPushJobStatusStoreClusterName())
+          .entrySet()) {
+        LOGGER.info("Sending push job details: {} to region: {} for: {}", value, entry.getKey(), key);
+        ControllerResponse response = entry.getValue()
+            .sendPushJobDetails(
+                key.storeName.toString(),
+                key.versionNumber,
+                getPushJobDetailsSerializer().serialize(null, value));
+        if (response.isError()) {
+          LOGGER.error(
+              "Unable to send push job details to region: {}, for key: {}, value: {} for error: {}",
+              entry.getKey(),
+              key,
+              value,
+              response.getError());
+          lastDualWriteError = response.getError();
+        } else {
+          // When A/A is enabled, we only need to write to one child region and it will be replicated to all regions.
```

Comment:
Do we need to do an AA check here then?

Reply:
Hmm what do you mean? Today, when AGG mode is enabled, RT is created in all the fabrics. So even today writing to child region is good. I want to let it write to child for a while (like a day) and then do the store property upgrade and create a new version and then daily push job report won't be broken during the transition. And then in phase 2, I remove the write to parent fabric, and set the A/A mode in the system init routine and make the integration test changes.